fails-first.user.js

```diff
@@ -27,13 +28,18 @@
                 skipped.push(child);
             } else if (child.querySelector('.octicon-x')) {
                 child.parentNode.removeChild(child);
+                const isRequired = Array.from(child.querySelectorAll('span')).some(el => el.textContent === 'Required');
+                if(isRequired) {
```

Comment:
I think you can do `child.innerText.contains('Required')` -- also I'm surprised eslint didn't rewrite `if(isRequired)` -> `if (isRequired)`